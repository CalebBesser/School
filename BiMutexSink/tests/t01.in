9 10
0 3
0 8
1 5
2 5
2 7
4 7
5 0
7 0
8 3
8 6
biMutexSink
