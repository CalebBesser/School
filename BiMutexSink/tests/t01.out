Running biMutexSink.
The graph is NOT bi-Mutex-Sink.
